spec/controllers/user_targets_controller_spec.rb

```diff
@@ -61,4 +62,90 @@
       end
     end
   end
+
+  describe '#create' do
+    before(:each) do
+      sign_in user
+    end
+
+    context 'with valid params' do
+      let(:new_target) { target_params('title', 50, topic.id, 0, 0) }
+      let(:target) { UserTarget.last }
+
+      before(:each) do
+        post :create, params: new_target, xhr: true
+      end
+
+      it 'responds HTTP 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'render create view' do
+        expect(response).to render_template('create')
+      end
+
+      it 'create a target with correct data' do
+        expect(target.title).to eq(new_target[:user_target][:title])
+        expect(target.latitude).to eq(new_target[:user_target][:latitude])
```

Comment:
Always try to just one `expect` per test when possible, that way you check attributes separately and the tests results provides more information.